sdk/servicebus/service-bus/test/stress/stress-test-resources.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
Looks like this is already being generated from bicep. Please check in the bicep file and not the json file (and optionally add a .gitignore entry so the json file stays excluded).